qlib/config.py

```
@@ -23,17 +23,21 @@
 from typing import TYPE_CHECKING
 
 from qlib.constant import REG_CN, REG_US, REG_TW
+from qlib.utils import hash_args, get_module_logger
 
 if TYPE_CHECKING:
     from qlib.utils.time import Freq
 
 
 class Config:
+    _global_state = {}  
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)